package.json

```diff
@@ -61,6 +61,7 @@
   "scripts": {
     "buildCore": "./node_modules/requirejs/bin/r.js -o build/core.build.json",
     "buildCompressor": "./node_modules/requirejs/bin/r.js -o build/compressor.build.json",
-    "test": "make tests"
```

Comment:
`make tests` needed to change to `make test` to match the task name in `Makefile`.